src/App.jsx

```diff
@@ -0,0 +1,30 @@
+import './App.css'
+
+function App() {
+
+  const handleClick = () =>{
+    window.alert("Clicked!")
+  }
```

Comment:
To improve the readability of the code you can put this code on a single line, use the concise body instead of the block body.
```suggestion
  const handleClick = () => window.alert("Clicked!")
```